test/unit/rerouting-test.js

```diff
@@ -381,4 +386,15 @@ describe('Connecting to a server that sends a re-routing information', function(
       connection.close();
     }
   });
+
+  it('Test if routing data capture the correct instance name value', async function() {
```

Comment:
How do you feel about moving this to a different test file where it fits better conceptually?